config/navigation.yaml

```diff
@@ -21,8 +22,11 @@ gate:
   post_radius: 0.7
 
 search:
-  stop_thresh: 0.6
+  stop_thresh: 0.5
   drive_fwd_thresh: 0.34
+  coverage_radius: 20
+  points_per_spiral: 8
```

Comment:
Seems like this isn't being used? Do we want to get rid of this and change segments_per_rotation to 8?